templates/email/default/other-reported.html

```diff
@@ -12,14 +12,22 @@
   [% start_padded_box | safe %]
   <h1 style="[% h1_style %]">Your report has been&nbsp;logged</h1>
   <p style="[% p_style %]">Your report to [% report.body %] has been logged on [% site_name %].</p>
+
+[% IF report.non_public %]
```

Comment:
Looks like the cobrand-templates check missed this, but this block and `view_url` should probably be added to the overridden templates in `tfl`, `hounslow`, and `buckinghamshire` (and maybe `fixamingata`?).